### DISCUSSION TITLE
Questions about Accessible Variables of Vehicle and Tire Power Loss

### DISCUSSION THREAD

**Discussion by Fantasydog05** [DLR-VSDC/IEEE-MVC-2023#6] [Q&A] [answered]
Hello, I would like to discuss some questions.
1. The vehicle states mentioned in the section IV.-> A. Input/Output Interfaces in the paper include the current velocities and accelerations. Are the motor velocities and the vehicle lateral acceleration included? Also like to ask if yaw rate can be used? It's commonly available from IMU.
2. The power losses on the tires are taken into consideration of the score. However, the load transfer during cornering, acceleration, and braking is neglected. It seems that the result of tire power loss by the baseline controller is always the smallest considering the slip ratio on the each wheel are always quite close, which is probably caused by the average distribution of the torques and the neglected load transfer. I would like to ask is the assumption of neglecting load transfer reasonable in the consideration of power loss on the tires?

**Comment by JBRDLR** [comment, marked as answer]
Hi,

Regarding 1: 
we decided to have the fixed in and output variables to the controller as in the bEMA and documented in the paper, but for experimental reasons you are free to to browse through the variable tree of the FMU and look at the desired variables. 

Regarding 2:
Yes, you are right. Since we have a planar mechanic we only have three DoF and therefore no tire load fluctuation are considered. 
For sake of complexity reduction we neglected this in the current version. Nevertheless, the torque vectoring / distribution might bring makes it possible to influence the slip on the different tires.

Best 
Jonathan

**Reply by Fantasydog05** [reply]
Thank you for your explanation. One more question, so is it correct that the requested torque from speed controller is also unknown to EMA?

**Reply by JBRDLR** [reply]
As shown in the presentation: 
![image](https://user-images.githubusercontent.com/110536277/209691243-daca254c-a3bf-4500-859c-1fa218a99954.png)
These are the inputs and outputs to the EMA.
[@rpinto](https://github.com/rpintodecastro), probably you can answer, whether or not it is allowed to use more in/outputs or we do make a certain "whishlist" available to the competitors?

**Reply by rpintodecastro** [reply]
We can extend the states available to the EMA. The lateral acceleration and torque/force requested by the speed controller can be included in the state vector. @Fantasydog05 would you like to access any other state?

**Reply by Fantasydog05** [reply]
Thank you so much. I would like to make final confirmation. So are the angular velocity of the motors and yaw rate mentioned previously not available? The mentioned variable names in the program are attached in the figure below.
![image](https://user-images.githubusercontent.com/87512543/209780738-9ed1e082-6113-4351-8209-28b7949baff0.png)

**Comment by JBRDLR** [comment]
Further discussion please refer to #8
